src/cockpit/channels/dbus.py

```diff
@@ -243,7 +244,9 @@ async def do_call(self, call, message):
             # watch processing, wait for that to be done.
             async with self.watch_processing_lock:
                 # TODO: stop hard-coding the endian flag here.
-                self.send_message(reply=[reply], id=cookie, flags="<" if flags is not None else None)
+                self.send_message(reply=[reply], id=cookie,
+                                  flags="<" if flags is not None else None,
+                                  type=type if type is not None else None)
```

Comment:
`type if type is not None else None` is a very fancy way to say `type`.

Reply:
Indeed!  Fixed.